### PR TITLE
[FIX] Adapt the UI according to the permission to put RO when creating a room

### PR DESCRIPTION
**Pull request by Jeanstaquet**
## Proposed changes (including videos or screenshots)

If we set that the user can't activate the RO, he can still do it by creating a room. I corrected this

### CODE REVIEW

File: client/sidebar/header/CreateChannel.js [hunk 1]
```diff
@@ -70,6 +71,8 @@ const CreateChannel = ({
 
 	const canSave = useMemo(() => hasUnsavedChanges && !nameError, [hasUnsavedChanges, nameError]);
 
+	const canSetReadOnly = usePermission('set-readonly');
+	console.log(canSetReadOnly)
```
Comment (KevLehman) [inline client/sidebar/header/CreateChannel.js:75]: Remove the console.log pls

**Review by KevLehman — CHANGES_REQUESTED** [review]
The console.log :)

File: client/sidebar/header/CreateChannel.js (hunk 1 above)
Comment (Jeanstaquet) [reply client/sidebar/header/CreateChannel.js:75]: Sorry, I missed it.

**Review by gabriellsh — CHANGES_REQUESTED** [review]
I tried fixing the conflicts and pushing to your branch, but that doesn't seem to be possible. Can you fix them? That file doesn't exist anymore, was renamed to be CreateChannel.tsx

**Pull request closed without merging**